instrumentation/opentelemetry_bandit/lib/opentelemetry_bandit.ex

```diff
@@ -44,6 +44,7 @@ defmodule OpentelemetryBandit do
     duration = measurements.duration
     end_time = :opentelemetry.timestamp()
     start_time = end_time - duration
+    resp_body_bytes = Map.get(measurements, :resp_body_bytes, 0)
```

Comment:
This is because in another app using Bandit 1.4 and this fix, encountered a situation where `:resp_body_bytes` was not set at all in measurements.